CMakeLists.txt

```diff
@@ -39,12 +39,13 @@ include(GNUInstallDirs)
 #-----------------------------------------------------------------------------
 set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -g3 -O0 -Wall -Wextra -Wno-missing-field-initializers -Wno-missing-braces -Wno-unused-result -Wno-ignored-attributes")
 set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -Werror -O2")
+string(REGEX REPLACE "O2 " "O3 " CMAKE_C_FLAGS_RELWITHDEBINFO "${CMAKE_C_FLAGS_RELWITHDEBINFO} -g -ggdb3" )
```

Comment:
`-g` is already present in `CMAKE_C_FLAGS_RELWITHDEBINFO`. Also, why not simply use `-g3` instead of the gdb specific `-ggdb3`?

Reply:
Thanks!

Reply:
Switched to `-g3` option here